bundler/lib/bundler/templates/newgem/Rakefile.tt

```diff
@@ -41,13 +41,23 @@ require "standard/rake"
 <% if config[:ext] -%>
 <% default_task_names.unshift(:compile) -%>
 <% default_task_names.unshift(:clobber) unless config[:ext] == 'rust' -%>
+<% if config[:ext] == 'rust' -%>
+require "rb_sys/extensiontask"
+
+task build: :compile
+
+RbSys::ExtensionTask.new(<%= config[:name].inspect %>) do |ext|
```

Comment:
@ianks 
Don't need to pass `gemspec` here? According to the document, it requires here. 
https://github.com/oxidize-rb/rb-sys/tree/main/gem#rbsysextensiontask  

If my understanding is correct, it is needed to check some data in the `gemspec`.
https://github.com/oxidize-rb/rb-sys/blob/dfc51c60bef89925bfded1713e9ec02c5dfb639d/gem/lib/rb_sys/extensiontask.rb#L139-L149

Reply:
Thanks. I understood the intent. I was curious because using `RbSys::ExtensionTask` without `gemspec` raises an error now.  It seems it's not intended so I will create a patch to `rb-sys`.

Reply:
Ty![ Released in v0.9.69](https://github.com/oxidize-rb/rb-sys/releases/tag/v0.9.69)